string path_to_file
---
bool sucess